tests/unit/defrag.cpp

```diff
@@ -0,0 +1,522 @@
+#define BOOST_TEST_DYN_LINK
+#define BOOST_TEST_MODULE DEFRAG library test
+
+#include <boost/test/unit_test.hpp>
+#include <boost/filesystem.hpp>
+
+#include <future>
```

Comment:
Why this include not near other std includes?

@shaitan Do we use the same code style as in elliptics?

Reply:
I copy pasted it from corruption test. Also I think we don't need this header actually 😄